README.md

```diff
@@ -136,6 +136,31 @@ process.on('SIGINT', function () {
 });
 ```
 
+#### Check accessibility
+Sometimes you want an easy way to check if the current system supports GPIOs programmatically, and mock some code if it doesn't. You can use `Gpio.accessible` for exactly this.
```

Comment:
The onoff documentation shouldn't use the words "you", your", "I", etc... The text here should be reformulated to avoid using these words.